nixos/modules/services/web-apps/fusion.nix

```diff
@@ -0,0 +1,144 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.fusion;
+in
+{
+  options.services.fusion = {
+    enable = lib.mkEnableOption "Fusion, a lightweight, self-hosted friendly RSS aggregator and reader";
+
+    host = lib.mkOption {
+      type = lib.types.str;
+      default = "0.0.0.0";
+      example = "[::1]";
+      description = "The address that Fusion should listen on.";
+    };
+
+    port = lib.mkOption {
+      type = lib.types.port;
+      default = 8080;
+      description = "The port that Fusion should listen on.";
+    };
+
+    passwordFile = lib.mkOption {
+      type = lib.types.path;
+      example = "/run/keys/fusion-password";
+      description = ''
+        A file containing the password for the Fusion web interface.
+
+        This should be a path pointing to a file within a secure directory and NEVER
+        in the Nix store (which is world-readable)!
```

Comment:
might it make sense to add a check to protect users from themselves on this [like here](https://github.com/NixOS/nixpkgs/pull/326306/files#diff-1eb0d5f14641b3640a20d71e7d4e1a9b9efcca7b152f5115f4029651bcff5817R193-R197)?
(maybe similar considerations could hold for other path options here.)

Reply:
I think I would prefer something like `lib.types.notStorePath` or something but that's not an essential improvement IMO